packages/cadl-ranch-specs/http/azure/core/lro/rpc/main.tsp

```diff
@@ -130,15 +130,15 @@ Expected response body:
 }
 ```
 """)
+@route("/same-poll-result")
 interface SamePollResult {
```

Comment:
nothing against this change but how is this changing for the new rule? Wasn't it just that it shouldn't have route at all?

Reply:
ok I see, the other operation is a custom copy of the core one so it doesn't complain about it.

I think the better choice here is just to suppress on the operation because that's by design we have this here

Reply:
The point of the new rule is to discourage the use of @route directly on an operation because someone might think that it will set the entire route of the operation, not just the fragment for the current container level.  The real fix is to put the route prefix on the container.  

If you want, I can just suppress instead, I'm fine with it either way.

Reply:
sorry thought you were already gone on leave, was just updating your branch already.

Yeah I think it is better for cadl-ranch as the route is really specific to how we want the test server to be structured and not really matter that it doesn't follow the standard(most cald-ranch specs are not azure.core)

Reply:
actually nvm, I'll revert you are probably right